dds/DCPS/RTPS/ICE/AgentImpl.cpp

```diff
@@ -215,6 +221,21 @@ void AgentImpl::check_invariants() const
   OPENDDS_ASSERT(expected == active_foundations);
 }
 
+void AgentImpl::add_address(const DCPS::NetworkInterface&,
+                            const ACE_INET_Addr&)
+{
+  for (EndpointManagerMapType::const_iterator pos = endpoint_managers_.begin(),
+         limit = endpoint_managers_.end(); pos != limit; ++pos) {
+    pos->second->network_change();
+  }
+}
+
+void AgentImpl::remove_address(const DCPS::NetworkInterface& interface,
+                               const ACE_INET_Addr& address)
+{
+  add_address(interface, address);
```

Comment:
Why does remove do an add?

Reply:
We have to poll for changes so the same code is executed in both.

Reply:
Should the method name be changed to reflect this?